universal_basic_income.markdown

```diff
@@ -7,32 +7,46 @@
-*   What's your reaction to hearing tech leaders like Elon Musk describe UBI as a "foregone conclusion"?
-*   Write a quick elevator pitch synthesizing what you've read from these articles: what is UBI, why is it entering policy debates, what do you think people should know about automation and its affect on our workforce?
-*   Do you think it's worthwhile to be educated about automation, its affect on our workforce, and policy solutions like UBI? What does it look like to be a responsible citizen in the face of such a large, inevitable transformation? Do you think you can really have an impact -- if not, why put energy into being informed?
+*   Do you think it's worthwhile to be educated about automation, its affect on our workforce, and policy solutions like UBI? What does it look like to be a responsible citizen in the face of such a large, inevitable transformation? Do you think you 
+* If you had a UBI, would syou still work(full-time, part-time, not at all?)
 
+### Related / Addendum Readings
 
-### Readings
+Interested in more? Check out the following:
 
+* [The One Thing Politicians Are Too Scared To Talk About](https://medium.freecodecamp.com/bill-gates-and-elon-musk-just-warned-us-about-the-one-thing-politicians-are-too-scared-to-talk-8db9815fd398#.7uibno4wd)
 *   ["About half of Americans support giving residents up to $2000 a month when robots take their jobs"](http://www.cnbc.com/2016/12/19/about-half-of-americans-support-giving-residents-up-to-2000-a-month-when-robots-take-our-jobs.html) (CNBC: Dec 19, 2016).
 *   ["Could universal basic income start with local and state legislation?"](https://medium.com/economicsecproj/could-universal-basic-income-start-with-local-and-state-legislation-5a3b60abd914#.75gotsomm) (Economic Security Project: Dec 19, 2016)
 *   ["If manufacturing jobs can't come back, what happens next?""](http://www.marketplace.org/2017/01/19/elections/big-promise/future-erie-will-jobs-come-back) (Marketplace.org: January 19, 2017)
 
-### Related / Addendum Readings
+## Reflection (5 min to write in journal)
 
-Interested in more? Check out the following:
+* Summarzie your thoughts about the video and UBI (answer above questions)
 
-* [The One Thing Politicians Are Too Scared To Talk About](https://medium.freecodecamp.com/bill-gates-and-elon-musk-just-warned-us-about-the-one-thing-politicians-are-too-scared-to-talk-8db9815fd398#.7uibno4wd)
+## Line-up (3 min to line up in hot/cold order of for/against UBI)
+
+## First Round Discussions  (15 min)
+
+* In your assigned groups (similar opinion) discuss why you are for UBI or against it.
+
+## Group change (3 min)
 
-## Discuss (25 min)
+## Second Round of Discussions (15 min)
 
-In groups of 3-4, discuss your answers to the questions above.
+* (number people 1's and 2's, 2's will change groups
+* In your assigned groups (mixed opinions) discuss why you are for UBI or against it.
```

Comment:
I like the "debate" style of seeing and hearing both sides of this in order to come to a clear opinion.